Add `Expert Global Taiko Showdown 2022` wiki article (#7649)
ppy/osu-wiki: wiki/Tournaments/GTS/EGTS_2022/en.md

---
tags:
  - EGTS2022
  - EGTS
  - GTS
---

# Expert Global Taiko Showdown 2022

![EGTS 2022 logo](img/logo.jpg)

> The **Expert Global Taiko Showdown 2022** (***EGTS 2022***) is a worldwide double-elimination osu!taiko tournament hosted by ::{ flag=FR }:: [\_yosh](https://osu.ppy.sh/users/7157133), ::{ flag=FR }:: [Arrival](https://osu.ppy.sh/users/1694000), and ::{ flag=FR }:: [Kasumi-sama](https://osu.ppy.sh/users/6177263). A tournament targeted towards top osu!taiko players, with no rank limits. It is the third instalment of the Expert Global Taiko Showdown and part of the Global Taiko Showdown series.

Reviewer: ```suggestion
The **Expert Global Taiko Showdown 2022** (***EGTS 2022***) is a worldwide double-elimination osu!taiko tournament hosted by ::{ flag=FR }:: [\_yosh](https://osu.ppy.sh/users/7157133), ::{ flag=FR }:: [Arrival](https://osu.ppy.sh/users/1694000), and ::{ flag=FR }:: [Kasumi-sama](https://osu.ppy.sh/users/6177263). The tournament is targeted for the top osu!taiko players, with no rank limits. It is the third instalment of the Expert Global Taiko Showdown and part of the Global Taiko Showdown series.
```

| Designer | ::{ flag=MY }:: [\[Zeth\]](https://osu.ppy.sh/users/9912966), ::{ flag=MY }:: [Jerry](https://osu.ppy.sh/users/605973), ::{ flag=PH }:: [OsuMe65](https://osu.ppy.sh/users/852867) |
| Translator | ::{ flag=FR }:: [\_yosh](https://osu.ppy.sh/users/7157133), ::{ flag=JP }:: [Noko\_BSF](https://osu.ppy.sh/users/3811831), ::{ flag=DE }:: [Sayira](https://osu.ppy.sh/users/7253958), ::{ flag=NL }:: [TaikoMom](https://osu.ppy.sh/users/9086438), ::{ flag=CN }:: [WonderMan](https://osu.ppy.sh/users/2221895) |
| Developer | ::{ flag=US }:: [Chupalika](https://osu.ppy.sh/users/1926383), ::{ flag=US }:: [Cychloryn](https://osu.ppy.sh/users/6921736), ::{ flag=NL }:: [oliebol](https://osu.ppy.sh/users/2756335), ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454) |
| Wiki editor | ::{ flag=ID }:: [fajar13k](https://osu.ppy.sh/users/7100002) |

## Links

- [Discord server](https://discord.gg/3mGC3HB)
- [EGTS website](https://gtsosu.com/egts/home)
- Livestream
  - [GTSosu](https://www.twitch.tv/gtsosu)
  - [GTSosu\_b](https://www.twitch.tv/gtsosu_b)

## Participants

 *Participants will be announced soon after the screening phase has ended.*

## Mappools

>  *Each mappools will be announced every 7 days (usually Sunday) before the next stage's match timeline.*

Reviewer: ```suggestion
 *Every mappool will be announced 7 days (usually Sunday) before the next stage's match timeline.*
```

## Ruleset

### General rules

1. This tournament is a **1v1 double-elimination tournament**.
2. Players will first go through Qualifiers, then continuing to the double-elimination bracket.
3. The scoring system used will be **Score V2**.

> 4. Any players are allowed to sign up since there are no rank limits applied.

Reviewer 2: ```suggestion
4. Any player is allowed to sign up due to no rank limit.
```

8. The ForceMod pool is played with ForceMod activated. Every player has to pick Hidden, HardRock, or both
9. The Tiebreaker is played under ForceMod conditions, but it is not necessary to have a mod activated.
10. The mappool for each stage will be showcased a week beforehand.

### Scheduling instructions

1. Every stage will be held **during the weekend**. It is, however, allowed to schedule it on another day, if no good time is found during the weekend and a referee is available.
2. Matches are allowed to overlap if referees are available.
3. A best effort will be made to fit schedules with applicable timezones. The schedule for each stage will be released the weekend prior to it.
4. Reschedules will only be considered if both players agree on a time. This needs to be done and notified to the tournament staff before **Thursday at 23:59 UTC** in the particular week when the match takes place.

### Match regulations

1. A private lobby will be created 10 minutes before a referee starts the match. Both players will be invited into the room before match time.
2. If no staff or referee is available, the match will be postponed.

> 3. If one of the player's coming late, the maximum time the match can be postponed is **10 minutes**.

Reviewer 2: ```suggestion
3. If one of the players is late, the maximum time the match can be postponed is **10 minutes**.
```
